### PR TITLE
CE: Mention that you can use ValueChangeMode for eagerness

### PR DESCRIPTION
**Pull request by Peppe**

### CODE REVIEW

File: articles/ce/collaboration-binder.asciidoc [hunk 1]
```diff
@@ -169,3 +169,16 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
 generic type for deserializing the value.
 Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
 you need to implement custom (de)serializer functions anyway.
+
+== Propagate Values Eagerly in Text Fields
+On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field. 
+However, in some use cases you might want to send the value of the field more eagerly. 
+You can configure how eagerly the data should be sent through the text fields own API, `TextField.setValueChangeMode(ValueChangeMode);`.
```
Comment (jouni) [inline articles/ce/collaboration-binder.asciidoc:176]:
```suggestion
You can configure how eagerly Text Field sends data through its own API, using `TextField.setValueChangeMode(ValueChangeMode);`.

pass:[<!-- vale Vaadin.Weasel = YES -->]
```

File: articles/ce/collaboration-binder.asciidoc [hunk 2]
```diff
@@ -169,3 +169,16 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
 generic type for deserializing the value.
 Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
 you need to implement custom (de)serializer functions anyway.
+
+== Propagate Values Eagerly in Text Fields
```
Comment (jouni) [inline articles/ce/collaboration-binder.asciidoc:173]:
```suggestion
pass:[<!-- vale Vaadin.Weasel = NO -->]

== Propagate Values Eagerly in Text Fields
```

File: articles/ce/collaboration-binder.asciidoc [hunk 3]
```diff
@@ -169,3 +169,20 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
 generic type for deserializing the value.
 Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
 you need to implement custom (de)serializer functions anyway.
+
+pass:[<!-- vale Vaadin.Weasel = NO -->]
+
+== Propagate Values Eagerly in Text Fields
+On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field.
```
Comment (pekam) [inline articles/ce/collaboration-binder.asciidoc:176]:
```suggestion
On text fields, the default and recommended mode for propagating values from one user to others is when the user blurs the field or presses the enter key.
```

File: articles/ce/collaboration-binder.asciidoc [hunk 4]
```diff
@@ -169,3 +169,20 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
...
+On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field. 
+However, in some use cases you might want to send the value of the field more eagerly. 
+You can configure how eagerly Text Field sends data through its own API, using `TextField.setValueChangeMode(ValueChangeMode);`.
+
+pass:[<!-- vale Vaadin.Weasel = YES -->]
+For example, to instantly send each keystroke to other users, you would do the following:
+
+[source, java]
+----
+textField.setValueChangeMode(ValueChangeMode.EAGER);
```
Comment (pekam) [inline articles/ce/collaboration-binder.asciidoc:185]:
```suggestion
TextField textField = new TextField();
textField.setValueChangeMode(ValueChangeMode.EAGER);
```

File: articles/ce/collaboration-binder.asciidoc [hunk 5]
```diff
@@ -169,3 +169,20 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
 generic type for deserializing the value.
 Note that if the element type is not supported by Collaboration Engine (e.g. `CheckboxGroup<Person>`),
 you need to implement custom (de)serializer functions anyway.
+
+pass:[<!-- vale Vaadin.Weasel = NO -->]
+
+== Propagate Values Eagerly in Text Fields
+On text fields, the default and recommended mode for propagating values from one user to others are when the user blurs the field. 
+However, in some use cases you might want to send the value of the field more eagerly. 
+You can configure how eagerly Text Field sends data through its own API, using `TextField.setValueChangeMode(ValueChangeMode);`.
```
Comment (pekam) [inline articles/ce/collaboration-binder.asciidoc:178]: The first sentence seems redundant.
The API exists on most fields, so I would be more generic here and leave `TextField` to the example code.
```suggestion
You can configure how eagerly the field sends data through its own API, using the `setValueChangeMode(ValueChangeMode)` method.
```

File: articles/ce/collaboration-binder.asciidoc [hunk 6]
```diff
@@ -169,3 +169,20 @@ This is necessary, because `CollaborationBinder` can't automatically infer the
...
+
+pass:[<!-- vale Vaadin.Weasel = YES -->]
+For example, to instantly send each keystroke to other users, you would do the following:
+
+[source, java]
+----
+textField.setValueChangeMode(ValueChangeMode.EAGER);
+----
+
+Modes like `ValueChangeMode.LAZY` and `ValueChangeMode.TIMEOUT` can also be used together with `textField.setValueChangeTimeout(int);` to reduce the amount of traffic.
```
Comment (pekam) [inline articles/ce/collaboration-binder.asciidoc:188]:
```suggestion
Modes like `ValueChangeMode.LAZY` and `ValueChangeMode.TIMEOUT` can also be used together with the `setValueChangeTimeout(int)` method to reduce the amount of traffic.
```

**Review by pekam — CHANGES_REQUESTED** [review]

**Review by pekam — APPROVED** [review]

**Review by jouni — APPROVED** [review]

**Pull request merged**